README.md

```diff
@@ -1,3 +1,32 @@
 # pyper
 
-Pyper is a Python package that offers powerful configuration management capabilities like the Viper package in Golang.
+Pyper is a Python package that offers powerful configuration management
```

Comment:
Not sure I would use the word "powerful" here, but can't explain exactly why. Perhaps because the term is subjective? Perhaps because it sounds like a marketing term?

Viper describes itself as "Viper is a complete configuration solution for Go applications". "complete" is also subjective, but somehow feels less like a fluffy marketing term.

Reply:
I agree. I also had an uncomfortable feeling when GitHub Copilot put this description in. I will reformulate it.